$MirOS: ports/comms/linphone/patches/patch-osipua_src_utils_c,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
from FreeBSD:       Revision 1.1, Thu Oct 23 03:29:23 2003 UTC by edwin
--- osipua/src/utils.c.orig	Wed Oct  6 00:28:29 2004
+++ osipua/src/utils.c	Wed Oct  6 00:29:01 2004
@@ -383,6 +383,7 @@ int guess_local_address(char *address_to
 		abort();
 		return -1;
 	}
+	((struct sockaddr_in *)(res->ai_addr))->sin_port=htons(9999);
 	err=connect(sock,res->ai_addr,res->ai_addrlen);
 	if (err<0) {
 		osip_trace(OSIP_ERROR,("Error in connect: %s\n",strerror(errno)));
